plugins/azure/virtualmachines/instantRestoreRetention.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
             default: '5'
         }
     },
+    realtime_triggers: ['microsoftcompute:virtualmachines:write', 'microsoftcompute:virtualmachines:delete', 'microsoftrecoverservices:valuts:write', 'microsoftrecoverservices:valuts:delete', 'microsoftrecoveryservices:vaults:backupfabrics:protectioncontainers:protecteditems:write'],
```

Comment:
```suggestion
    realtime_triggers: ['microsoftcompute:virtualmachines:write', 'microsoftcompute:virtualmachines:delete', 'microsoftrecoveryservices:vaults:write', 'microsoftrecoveryservices:vaults:delete', 'microsoftrecoveryservices:vaults:backupfabrics:protectioncontainers:protecteditems:write'],
```
vaults is misspelled and need to verify 'recoveryservices'